docs/meshcloud.customer.md

```diff
@@ -3,29 +3,27 @@ id: meshcloud.customer
 title:  meshCustomer
 ---
 
-## What is a meshCustomer?
-
-A meshCustomer is automatically created for you when you [sign up](meshcloud.profile.md) for a new account. A meshCustomer usually represents your company or department. Within a meshCustomer you manage multiple [projects](meshcloud.project.md), your users and their access rights and maintain general data like addresses and payment methods.
+A meshCustomer usually represents a product team or department. Within a meshCustomer you manage multiple [meshProjects](meshcloud.project.md), your users and their access rights and maintain general data like addresses and payment methods.
 
 ## Customer Settings
 
 General information of a meshCustomer (like its name) can be edited here. The customer identifier is also shown here, but it can never be changed, as it is used as an immutable identifier of the meshCustomer for its representation in the different cloud platforms.
 
-If applicable in your meshcloud installation, also a Tax ID can be defined here for your meshCustomer.
+Depending on the requirements of your organization, you may be able to edit additional customer metadata.
```

Comment:
You can customize the customer according to your needs by setting and editing additional customer metadata.